presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -138,6 +138,7 @@
     public static final String OPTIMIZED_REPARTITIONING_ENABLED = "optimized_repartitioning";
     public static final String AGGREGATION_PARTITIONING_MERGING_STRATEGY = "aggregation_partitioning_merging_strategy";
     public static final String LIST_BUILT_IN_FUNCTIONS_ONLY = "list_built_in_functions_only";
+    public static final String LEGACY_TASK_PRIORITY_TRACKING = "legacy_task_priority_tracking";
```

Comment:
let's not call this "legacy" since  such scheduling policy might be reasonable for interactive workload . Maybe just a enum about "TASK_FAIR" vs. "QUERY_FAIR" , tuned for interactive/batch use case :)